.circleci/config.yml

```diff
@@ -78,3 +78,32 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+
+      - architect/push-to-app-collection:
+          context: "architect"
+          name: push-starboard-exporter-to-aws-app-collection
+          app_name: "starboard-exporter"
+          app_collection_repo: "aws-app-collection"
+          requires:
+            - push-starboard-exporter-to-control-plane-catalog
+            - push-starboard-exporter-to-aliyun
```

Comment:
Requiring this here so that we don't block pushing to the catalog waiting for Aliyun, but we don't add it to the collection until it is also ready in China